meta/documents/changelog_de.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes für Elastic Export Geizhals.de
 
+## v1.0.5 (2017-05-15)
+
+### Behoben
+- Der Bestand wird nun korrekt ausgewertet.
```

Comment:
If you have multiple changes you should adjust your changelogs to be equal with the others. Just that it looks like that the changelogs are made from one person with the same pattern ;)